Sources/HummingbirdJobs/JobQueueHandler.swift

```diff
@@ -15,6 +15,45 @@
 import Hummingbird
 import Logging
 
+/// Object handling a single job queue
+public class HBJobQueueHandler {
+    public init(queue: HBJobQueue, numWorkers: Int, eventLoopGroup: EventLoopGroup, logger: Logger) {
+        self.queue = queue
+        self.workers = (0..<numWorkers).map { _ in
+            HBJobQueueWorker(queue: queue, eventLoop: eventLoopGroup.next(), logger: logger)
+        }
+        self.eventLoop = eventLoopGroup.next()
+    }
+
+    /// Push Job onto queue
+    /// - Returns: Queued job information
+    @discardableResult public func enqueue(_ job: HBJob, on eventLoop: EventLoop) -> EventLoopFuture<JobIdentifier> {
```

Comment:
What does the `eventLoop` argument do?

Reply:
Is that where the (database) driver gets called from?

Reply:
It could be adding the job to a Redis/Fluent database